dashboard/app/views/layouts/application.html.haml

```diff
@@ -8,6 +8,9 @@
     - managed_test_server = "#{CDO.running_web_application? && CDO.test_system?}"
     - title = @page_title ? "#{@page_title} - #{appname}" : appname
     %title= title
+    -# make sure robots do not index non production environments or disallowed routes
+    -if !rack_env?(:production) || SharedConstants::DISALLOWED_ROUTES.any? {|r| request.fullpath.include?(r)}
```

Comment:
seems a little safer to check that the string starts with rather than just includes?